src/App.js

```diff
@@ -155,7 +155,15 @@ const App = () => {
           <Route path={`projects/:projectName/${PROJECT_MONITOR}`} element={<ProjectMonitor />} />
 
           {/*TODO: the Alerts Route will be updated with ML-8368 */}
-          <Route path="projects/:id/alerts" element={<ProjectsAlerts />} />
+          {[
+            'projects/:id/alerts',
+            'projects/:id/alerts/:project/:alertName/:entityName/:id/:tab',
```

Comment:
why do you add two similar routes?

Reply:
Likely from the rebase. Removed, thanks.